common/prometheus.go

```diff
@@ -0,0 +1,142 @@
+// Copyright © 2018 Barthelemy Vessemont
+// GNU General Public License version 3
+
+package common
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	IndexProbeStatus = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "es_index_probe_status",
+			Help: "Indicate index probe status (green is 1, yellow is 1 and red is 2)",
+		},
+		[]string{"cluster", "index"},
+	)
+
+	ClusterDurabilityDocumentsCount = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "es_cluster_durability_documents_count",
+			Help: "Reports number of documents count in durability index",
+		},
+		[]string{"cluster"})
+
+	ClusterSearchDocumentsHits = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "es_cluster_search_documents_hits",
```

Comment:
`es_cluster_durability_search_documents_hits`  or `es_cluster_search_durability_documents_hits` would make it easier to understand the purpose of the metric